_integration-schemas/outreach/v1/mailings.md

```diff
@@ -0,0 +1,155 @@
+---
+tap: "outreach"
+version: "1"
+
+name: "mailings"
+doc-link: "https://api.outreach.io/api/v2/docs#mailing"
+singer-schema: "https://github.com/singer-io/tap-outreach/blob/master/tap_outreach/schemas/mailings.json"
+description: |
+  The {{ table.name }} table is a representation of a platform-related table.
```

Comment:
I see this description is based on Outreach's API docs (which is fine), but it reads a bit weird. Should it be `platform-related email` instead of `platform-related table`?